server.js

```diff
@@ -25,6 +25,34 @@ app.use(express.static(path.join(__dirname, 'public')))
 
 io.on('connection', handleSocketConnection)
 
+function startGame() {
+  console.log(`${currentRoom} will start a game in ${START_GAME_DELAY} sec`)
+  clearTimeout(countDownTimer)
+  countDownTimer = setTimeout(function startAGame() {
+    if (users.count() >= MIN_PLAYER_COUNT) { // if nobody has left lobby
+      io.to(currentRoom).emit('stateChange', 'game')
+      console.log(`${currentRoom} started a game!`)
+      currentRoom = generateRoomName()
+      users.clear()
+      io.emit('lobbyUpdate', users.list())
+    }
+  }, START_GAME_DELAY * ONE_SECOND + DELAY_BUFFER_IN_MS)
+  io.to(currentRoom).emit('countdown', START_GAME_DELAY)
+}
+
+function assignRoles(currentRoom) {
+  io.in(currentRoom).clients((err, clients) => {
```

Comment:
I think it depends on the situation...
Here I think 'in' makes sense. You are saying io. in current room . clients. In other situations I think 'to' makes sense.